Source/SwiftLintFramework/Rules/Lint/TestCaseAccessibilityRule.swift

```diff
@@ -41,7 +41,9 @@ public struct TestCaseAccessibilityRule: Rule, OptInRule, ConfigurationProviderR
 
     private func testClasses(in file: SwiftLintFile) -> [SourceKittenDictionary] {
         return file.structureDictionary.substructure.filter { dictionary in
-            dictionary.declarationKind == .class && dictionary.inheritedTypes.contains("XCTestCase")
+            dictionary.declarationKind == .class &&
+                (dictionary.inheritedTypes.contains("XCTestCase") ||
+                 dictionary.inheritedTypes.contains(where: configuration.xctestcaseSubclasses.contains))
```

Comment:
By having `XCTestCase` already contained in `configuration.xctestcaseSubclasses` we could write it like:
```suggestion
            dictionary.declarationKind == .class && configuration.xctestcaseSubclasses.intersection(dictionary.inheritedTypes).isNotEmpty
```